components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/BrandingPreferenceRetrievalClient.java

```diff
@@ -87,6 +89,8 @@ public JSONObject getPreference(String tenant, String type, String name, String
                 if (StringUtils.isNotBlank(locale)) {
                     uriBuilder.addParameter(RESOURCE_LOCALE_URL_SEARCH_PARAM, locale);
                 }
+
+                uriBuilder.addParameter(RESTRICT_TO_PUBLISHED_URL_SEARCH_PARAM, TRUE);
```

Comment:
Also don't we need to consider this param for getCustomTextPreference method?

Reply:
Fix for text customizations is tracked by https://github.com/wso2/product-is/issues/20858